src/audio/host.c

```diff
@@ -108,7 +108,7 @@ static void host_update_position(struct comp_dev *dev, uint32_t bytes)
 		hd->local_pos = 0;
 
 	/* NO_IRQ mode if host_period_size == 0 */
-	if (dev->params.host_period_bytes != 0) {
+	if (dev->params.period_irq != 0) {
```

Comment:
fix comment above?

Reply:
@plbossart, ahh I overlooked it. Thanks for spotting it. I have updated both sides, can we merge it together now?